mcs/class/monodoc/Monodoc/providers/EcmaDoc.cs

```diff
@@ -578,16 +602,19 @@ internal static string MakeOperatorSignature (XElement member, out string member
 
 		static XElement ExtractClassSummary (XDocument typeDoc)
 		{
-			string name = typeDoc.Root.Attribute("Name").Value;
-			string fullName = typeDoc.Root.Attribute("FullName").Value;
-			string assemblyName = typeDoc.Root.Element("AssemblyInfo") != null ? typeDoc.Root.Element("AssemblyInfo").Element("AssemblyName").Value : string.Empty;
-			var docs = typeDoc.Root.Element("Docs");
-			var summary = docs.Element("summary") ?? new XElement("summary");
-			var remarks = docs.Element("remarks") ?? new XElement("remarks");
+			string name = typeDoc.Root.Attribute ("Name").Value;
```

Comment:
Ok ... I will be very much more careful with this in the future. Thanks for the explanation :)